wp-content/themes/init_theme_name/theme/class-theme.php

```diff
@@ -84,7 +88,8 @@ public function enqueue_scripts() {
       wp_register_script( $this->theme_name . '-webfont', get_template_directory_uri() . '/skin/public/scripts/vendors/webfont.1.6.26.min.js', array(), '1.6.26' );
```

Comment:
What is this for?

Reply:
asynchronously loading fonts via js using google web fonts script.
In the future I plan to check if this can be set up using PostCss

Reply:
:+1: